### PR TITLE
React 16 Support Fix For PropType and CreateClass 

### PR DESCRIPTION
**Pull request by hackable**
Fixes PropTypes and CreateClass For React 16

### CODE REVIEW

**Comment by dnacenta** [comment]
Since these libraries have been updated now to solve that problem I suggest to link their new versions to the package.json instead of the forked ones.

react-native-lightbox - 0.7.0
react-native-parsed-text - 0.0.19
react-native-invertible-scroll-view - 1.1.0

File: package.json [hunk 1]
```diff
@@ -41,8 +41,8 @@
     "moment": "2.18.1",
     "prop-types": "15.5.10",
     "react-native-communications": "2.2.1",
-    "react-native-invertible-scroll-view": "git+https://github.com/brunocascio/react-native-invertible-scroll-view.git",
-    "react-native-lightbox": "git+https://github.com/brunocascio/react-native-lightbox.git",
+    "react-native-invertible-scroll-view": "^1.0.0",
+    "react-native-lightbox": "0.7.0",
     "react-native-parsed-text": "git+https://github.com/brunocascio/react-native-parsed-text.git",
```
Comment (dnacenta) [inline package.json]: version 0.0.19 of react-native-parsed-text solved the problem

**Comment by alexbowers** [comment]
Can we get some action on this please @FaridSafi , it'd be good to get rid of the warnings.

File: package.json [hunk 2]
```diff
@@ -41,9 +41,9 @@
     "moment": "2.18.1",
     "prop-types": "15.5.10",
     "react-native-communications": "2.2.1",
-    "react-native-invertible-scroll-view": "1.0.0",
-    "react-native-lightbox": "oblador/react-native-lightbox#c84a8543d4511fe6a44c3d7820747c9c1bddd875",
-    "react-native-parsed-text": "0.0.18",
+    "react-native-invertible-scroll-view": "^1.0.0",
+    "react-native-lightbox": "0.7.0",
+    "react-native-parsed-text": "git+https://github.com/brunocascio/react-native-parsed-text.git",
```
Comment (alexbowers) [inline package.json]: Should this not just be `^0.0.19`

**Comment by brunocascio** [comment]
@alexbowers @hackable I've just updated my PR: https://github.com/FaridSafi/react-native-gifted-chat/pull/523. Now it is using original sources.

Thanks!

**Comment by brunocascio** [comment]
Fixed via: https://github.com/FaridSafi/react-native-gifted-chat/pull/523

**Comment by alexbowers** [comment]
@brunocascio Can this be tagged please? :D

**Comment by brunocascio** [comment]
@alexbowers Done! 💃

**Comment by alexbowers** [comment]
Can you please also update NPM?

https://www.npmjs.com/package/react-native-gifted-chat still lists  0.2.7

**Comment by brunocascio** [comment]
@alexbowers 

Sorry, I forgot that!  Updated :D

**Comment by alexbowers** [comment]
Thanks :)

**Pull request closed without merging**